multimodal/src/autogluon/multimodal/data/preprocess_dataframe.py

```diff
@@ -655,7 +655,7 @@ def transform_document(
     def transform_numerical(
         self,
         df: pd.DataFrame,
-    ) -> Tuple[Dict[str, NDArray], None]:
+    ) -> Tuple[Dict[str, NDArray[np.float64]], None]:
```

Comment:
Do we need to specify precision here? I'm curious if there will be cases where float32 is used.

Reply:
I think this can be removed.